lang/de.json

```diff
@@ -634,6 +635,9 @@
     "Lay out user": "Auslage Benutzer",
     "Leave empty to generate a new Product Number.": "Leer lassen, um eine neue Produktnummer zu erzeugen.",
     "Ledger Account": "Buchungskonten",
+    "Ledger Accounts": "Buchungskonten",
+    "Ledger Account Type": "Buchungskonten-Typ",
```

Comment:
Singular

Reply:
in English heisst Ladger Accounts - daher brauche ich Plural